### DISCUSSION TITLE
How to update Blender without another install?

### DISCUSSION THREAD

**Discussion by Xiarno** [Victor-IX/Blender-Launcher-V2#132] [Q&A] [answered]
So I installed Blender 4.2.0 LTS through the Blender launcher. I wanna stay on LTS. Now Blender released 4.2.1 LTS and I want to update to it, without having a whole another install of Blender. So I keep all my things.

How do I achieve that with the Blender Launcher please?
Thanks!

**Comment by Victor-IX** [comment, marked as answer]
You just need to download the new version and remove the old one. Blender minor versions share the same preference folder. 
Note that it's currently not possible to download stable builds through the Blender Launcher at this time; see [issue 131](https://github.com/Victor-IX/Blender-Launcher-V2/issues/131#issuecomment-2288922757).

**Reply by Xiarno** [reply]
But I can download it through the daily tho?
![16507_Blender_Launcher_nt9ziOb27d](https://github.com/user-attachments/assets/b8d7a2d0-f669-480e-b6b9-52cafd4a1434)

**Reply by Victor-IX** [reply]
Yes, but it will only be available for a short period of time. When a build is released (stable), it's usually removed from the daily website a few days after the release.

**Reply by Xiarno** [reply]
> Yes, but it will only be available for a short period of time. When a build is released (stable), it's usually removed from the daily website a few days after the release.

But for now, I can still use it while it's there, right?

**Reply by zeptofine** [reply]
You can download it, and it will stay on your system. You can use it as long as you want. but it will be replaced in the daily downloads every day

**Reply by zeptofine** [reply]
I have a tendency to just keep downloading the 4.3.0 dailies for weeks then go through a purging spree of old builds

**Reply by Victor-IX** [reply]
4.2.1 will be the same build every day as it is no longer in development

**Comment by fabian-madco** [comment]
Shouldn't the version 4.2.1 LTS be available in the stable download by now?

**Reply by Victor-IX** [reply]
https://github.com/Victor-IX/Blender-Launcher-V2/issues/131 for more info
